From: Lukas Bulwahn <lukas.bulwahn@redhat.com>
Subject: MAINTAINERS: adjust file entry in UUID HELPERS
Date: Wed, 17 Dec 2025 06:39:07 +0100

Commit e61d3add3e7f ("lib/tests: convert test_uuid module to KUnit") moves
lib/test_uuid.c to lib/tests/uuid_kunit.c, but misses to adjust the file
entry in the MAINTAINERS section UUID HELPERS.

Adjust the file entry in that section after this file movement.

Link: https://lkml.kernel.org/r/20251217053907.2778515-1-lukas.bulwahn@redhat.com
Signed-off-by: Lukas Bulwahn <lukas.bulwahn@redhat.com>
Acked-by: David Gow <davidgow@google.com>
Acked-by: Ryota Sakamoto <sakamo.ryota@gmail.com>
Cc: Andriy Shevchenko <andriy.shevchenko@linux.intel.com>
Cc: Brendan Higgins <brendan.higgins@linux.dev>
Cc: Geert Uytterhoeven <geert@linux-m68k.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 MAINTAINERS |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/MAINTAINERS~lib-tests-convert-test_uuid-module-to-kunit-fix
+++ a/MAINTAINERS
@@ -27245,7 +27245,7 @@ R:	Andy Shevchenko <andriy.shevchenko@li
 L:	linux-kernel@vger.kernel.org
 S:	Maintained
 F:	include/linux/uuid.h
-F:	lib/test_uuid.c
+F:	lib/tests/uuid_kunit.c
 F:	lib/uuid.c
 
 UV SYSFS DRIVER
_
